blip-project/src/components/Page/Src/api/MeetingEndApi.jsx

```
@@ -0,0 +1,200 @@
+import axios from "axios";
+
+// 팀 ID를 정제하는 함수 추가
+const cleanTeamId = (teamId) => {
+  console.log("teamId", teamId);
+  if (typeof teamId === "string" && teamId.includes("create-")) {
+    const match = teamId.match(/create-(\d+)/);
+    return match && match[1] ? match[1] : teamId;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)